app/controllers/sessions_controller.rb

```diff
@@ -107,11 +107,9 @@ def create
           redirect_to profile_path,
                       alert: "That way to log in cannot be added because it is associated to an email address that is already in use!" # TODO i18n
         else
-          Rails.logger.fatal "IllegalState: OAuth data: #{request.env['omniauth.auth']}; " \
-                             "status: #{@handler_result.outputs[:status]}"
-          raise IllegalState, "SessionsCreate errors: #{@handler_result.errors.inspect
-                              }; Last exception: #{$!.inspect}; Exception backtrace: #{$@.inspect
-                              }; status: #{@handler_result.outputs[:status]}"
+          Rails.logger.fatal "IllegalState: OAuth data: #{request.env['omniauth.auth']
+                             }; status: #{@handler_result.outputs[:status]}"
```

Comment:
I think we want emails for unknown callback state (because that means we're not handling something)?

And invalid omniauth data -- seems like we want emails for that too so we can detect if we're not handling something correctly?